### DISCUSSION TITLE
need help with Venus Touch screen calibration

### DISCUSSION THREAD

**Discussion by Fraser86** [Louisvdw/dbus-serialbattery#441] [General]
Hey Guys and Girls. 

I am struggling with a small problem. 
I have had this before but not sure why but I can't seem to fix it. (I will explain)
After updating to Venus v3 my touch screen needs to be recalibrated. 
I had this before and managed to run the re-calibration script by turning the GUI to headless with this command 

Configure headless mode to disabled (enables the GUI on screen)
You need to reboot for this setting to take effect.

mv /etc/venus/headless /etc/venus/headless.off
sudo reboot now

In the past I reversed the headless and headless.off accordingly, in order to run the ts-calibrate. 

BUT NOW
when I enter ssh and use this above command it says file not found. 

This is probably something simple but can anyone help me to either get the GUI to switch or another way to run calibration of touchscreen, (p.s not an official pi touchscreen)

**Comment by vincegod** [comment]
Probably best to post this question in the Victron community forum as this place is for  dbus-serialbattery discussion and problems
